web3/providers/tester.py

```diff
@@ -48,8 +49,25 @@ def is_testrpc_available():
 )
 
 
+def static_return(value):
+    def inner(*args, **kwargs):
+        return {'result': value}
+    return inner
```

Comment:
This would probably be useful to other middlewares. Maybe it belongs in `web3/utils/formatters.py` or a new `web3/middleware/utils.py`?

Reply:
Done.  `web3.utils.formatters` for now.

Side note. I named that module and I'm growing to hate the terms formatter/formatting.  Wanting to move back towards the term `normalizer/normalizing` as it has more semantic meaning....